scripts/variantstore/docs/aou/AOU_DELIVERABLES.md

```diff
@@ -52,26 +50,32 @@
-   - This workflow transforms the data in the vet and ref_ranges tables into a schema optimized for VCF generation during the Extract step.
-   - It will need to be run twice, once with `control_samples` set to "true" (see [naming conventions doc](https://docs.google.com/document/d/1pNtuv7uDoiOFPbwe4zx5sAGH7MyxwKqXkyrpNmBxeow) for guidance on what to use for `extract_table_prefix` or cohort prefix, which you will need to keep track of for the `GvsExtractCallset` WDL); the default value is `false`.
+7. `GvsExtractAvroFilesForHail` workflow (Hail VDS output)    
+   - This workflow extracts the data in BigQuery and transforms it into Avro files in a Google bucket, incorporating the VQSR filter set data.  We suggest this method for callsets of more than 100k samples.
+   - The extracted Avro files will then be used as the input for the below notebook which can then be run to produce a Hail VDS
    - This workflow does not use the Terra Data Entity Model to run, so be sure to select the `Run workflow with inputs defined by file paths` workflow submission option.
-8. `GvsExtractCallset` workflow
-   - This workflow extracts the data in BigQuery and transforms it into a sharded joint called VCF incorporating the VQSR filter set data.  We will probably not run this on callsets of more than 100K samples.
-   - It also needs to be run twice, once with `control_samples` set to "true", and with the `filter_set_name` and `extract_table_prefix` from step 5 & 6.  Include a valid (and secure) "output_gcs_dir" parameter, which is where the VCF, interval list, manifest, and sample name list files will go.
+8. Run the VDS Extract using a notebook terminal and the python script `hail_gvs_import.py`
+   - This step creates a VDS based on the Avro files 
+   - Notebook provisioning suggestions can be found in the file: [AoU Delta VDS Cluster Configuration.md](vds/cluster/AoU Delta VDS Cluster Configuration.md)
+   - `hail_gvs_import.py` is the python script that needs to be run in the notebook-- we suggest running it directly in the terminal
+   - inputs to the python script are:
+     1. `--avro-path`: the directory path at which exported GVS Avro files are found in GCP
+     2. `--vds-path`: the desired output path to which the VDS should be written
+     3. `--temp-path`: a convenient path to temporary directory (we suggest the GCP bucket of the workspace that the notebook is in) 
+9. `GvsPrepareRangesCallset` workflow (Hail VDS output -- for callset stats)
+   - This workflow transforms the data in the vet tables into a schema optimized for VCF generation during the Extract step.
+   - It will need to be run once with `only_output_vet_tables` set to "true" (see [naming conventions doc](https://docs.google.com/document/d/1pNtuv7uDoiOFPbwe4zx5sAGH7MyxwKqXkyrpNmBxeow) for guidance on what to use for `extract_table_prefix` or cohort prefix, which you will need to keep track of for the callset stats); the default value is `false`.
```

Comment:
```suggestion
   - It will need to be run after the VDS has been successfully created with `only_output_vet_tables` set to "true" (see [naming conventions doc](https://docs.google.com/document/d/1pNtuv7uDoiOFPbwe4zx5sAGH7MyxwKqXkyrpNmBxeow) for guidance on what to use for `extract_table_prefix` or cohort prefix); the default value is `false`.
```